pom.xml

```diff
@@ -310,6 +310,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.release>8</maven.compiler.release>
         <maven.enforcer.requireJavaVersion>[11.0.10,12)</maven.enforcer.requireJavaVersion>
+        <project.build.outputTimestamp>1680115922</project.build.outputTimestamp>
```

Comment:
it seems that they used to add the ouputTimestamp, but now they stopped. This is commit they added it https://github.com/finos/legend-engine/commit/d118dc020a2b1d030ff4910d1d67e3ad6aa8efc1. When we merge with the latest master it will go away